### PR TITLE
Migrate samples to Swift 3

### PR DESCRIPTION
**Pull request by dhardiman**
Carthage seems to find this example project and fall over when the dependencies are checked using Xcode 8 as it’s not specifying the swift version. This appears to fix that. It does seem to build the framework target twice due to the cocoa pods usage for the example, but I don’t think that causes a problem. Perhaps in future the example project could link directly to the project rather than using CocoaPods to solve that, but probably not urgent.

### CODE REVIEW

**Comment by AliSoftware** [comment]
👌 Could you also add an entry in the CHANGELOG to credit yourself for this migration? Thx!

**Comment by dhardiman** [comment]
Sure, updated. Hope the format is ok above the previous release? Let me know if not and I'll fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # OHHTTPStubs — CHANGELOG
 
+* Migrate samples to Swift 3
```
Comment (AliSoftware) [inline CHANGELOG.md]: Could you add a full stop and 2 spaces at the end of that line so that the markdown formatting matches the other notes?

Comment (dhardiman) [inline CHANGELOG.md]: Updated

**Comment by AliSoftware** [comment]
Travis-CI builds are currently failing due to a bug in the latest version of `xcpretty`: https://github.com/supermarin/xcpretty/issues/241 — Waiting for the fix.

**Comment by AliSoftware** [comment]
Merged manually via ca8e9c295b859215b1ab8a9df8e0546a24e768e5

Thanks again!

**Pull request closed without merging**